### PR TITLE
Prioritize search

### PR DESCRIPTION
**Pull request by motrell**
Implemented the search tag boosting. Please have a look on it.

### CODE REVIEW

File: src/application/res/app.py [hunk 1]
```diff
@@ -41,13 +42,13 @@ class AdvancedEntryForm(FlaskForm):
         ('is_greater', 'is greater'), 
         ('is_smaller', 'is smaller')])
     value = StringField('Value')
+    weight = IntegerField('Weight', [validators.NumberRange(min=1, max=100, message="Weight must be between 1 and 100")],default=1)
```
Comment (motrell) [inline src/application/res/app.py:45]: @oninach this value `max=100` is chosen by us, right?

**Review by motrell — COMMENTED** [review]
Looks good! One question from myself remaining (see below), but it seems to work good!

**Review by oninach — APPROVED** [review]

**Review by leondaniel22 — APPROVED** [review]
I didn't test it properly but it looks very good on first glance! If you. tested it and it works as expected, we can merge it!

**Review by JMiltner97 — APPROVED** [review]

File: src/application/res/app.py (hunk 1 above)
Comment (motrell) [reply src/application/res/app.py:45]: done

**Pull request merged**